.vscode/settings.json

```diff
@@ -1,9 +1,11 @@
 {
-	"editor.insertSpaces": false,
-	"tslint.enable": true,
-	"typescript.tsc.autoDetect": "off",
-	"typescript.preferences.quoteStyle": "single",
-	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
-	}
-}
\ No newline at end of file
+  "editor.insertSpaces": false,
+  "tslint.enable": true,
+  "typescript.tsc.autoDetect": "off",
+  "typescript.preferences.quoteStyle": "single",

```

Comment:
does it have an effect of anything if we are using prettier anyway?

Reply:
not sure, I just left it there since it was already in there. But I'll remove it now that we have the prettier config, which should handle it